test/InterfaceStub.cpp

```diff
@@ -0,0 +1,202 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/** @file InterfaceStub.cpp
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ */
+
+#include <boost/test/unit_test.hpp>
+#include <libdevcore/CommonJS.h>
+#include "TestUtils.h"
+
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+using namespace dev::test;
+
+BOOST_FIXTURE_TEST_SUITE(InterfaceStub, ParallelInterfaceStubFixture)
+
+BOOST_AUTO_TEST_CASE(blocks)
+{
+	enumerateInterfaces([](Json::Value const& _json, dev::eth::InterfaceStub& _client) -> void
+	{
```

Comment:
When using `ParallelInterfaceStubFixture` this lamda function is invoked on multiple threads.
`InterfaceStubFixture` can be used to run the same code on a single thread